app/assets/stylesheets/z_custom.scss

```diff
@@ -0,0 +1,302 @@
+/* COLUMNS */
+
+.column, .columns{
+  padding: 20px 0px;
+}
+.container.staticpages {
+  max-width: 100%;
+}
+
+.inner-container{
+  max-width: 1200px;
+  margin: 0 auto;
+}
+/* FONTS */
+h1, h2, h3, h4, h5, h6, .main-nav{
+  font-family: 'Oswald', sans-serif;
+  color:#fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own line<br>Colon after property should be followed by one space